<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectsListDialog</class>
 <widget class="QDialog" name="ObjectsListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="labelLogo">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resource.qrc">:/res/Images/gas_stantion.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Список об'єктів</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frameObjects">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Фільтри</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="5">
          <widget class="QComboBox" name="comboBoxRegion"/>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Адреса(Назва):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Тармінал:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxClient"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Регіон:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Клієнт:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLineEdit" name="lineEditSearch"/>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="lineEditTerminalId">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="tableViewObjects">
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
